### DISCUSSION TITLE
Set `SVG_DIRS` with environ

### DISCUSSION THREAD

**Discussion by jasonleex** [xshapira/django-simple-svg#6] [Q&A] [answered]
I'm having difficulty configuring `SVG_DIRS` using environ. Could you provide guidance on the correct way to set this variable?

**Comment by xshapira** [comment, marked as answer]
To set the `SVG_DIRS` variable, you can first define your `ROOT_DIR` using the `environ.Path` like so:
```python
# Build paths inside the project like this: str(ROOT_DIR("subdir"))
ROOT_DIR = environ.Path(__file__) - 3
```
This will create a `Path` object representing the directory three levels above the current file. Note that the number of levels may vary depending on your file structure.

Once you have defined your `ROOT_DIR`, you can set `SVG DIRS` like this:
```python
SVG_DIRS = [str(ROOT_DIR("dist/img/svg"))]
```
This will set `SVG_DIRS` to a list containing the `svg` directory located in `dist/img` relative to the `ROOT_DIR`.
